api/scenario_api_simulator/src/scenario_api_simulator.cpp

```diff
@@ -51,13 +73,12 @@ bool ScenarioAPISimulator::isAPIReady()
 
 bool ScenarioAPISimulator::updateState()
 {
-  ros::spinOnce();
```

Comment:
I removed this since it doesn't seem sensible to me – why is this done? The node was just spinning, before the timer callback.

Reply:
Removing this makes sense to me.